src/classes/AccountDeDuper.cls

```diff
@@ -4,11 +4,17 @@ public with sharing class Account_DeDuper {
 Public AccountDe_Duper () {
 
 }
+public void badStuff() {}
+public void badStuff2() {}
+public void BADSTUFF3() {}
```

Comment:
* This method is empty, I suggest to remove it.
* Methods should use the `lowerCamelCase` naming convention. Please rename accordingly.